admin/publish-release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
We should decide on the "official" version of these files, and include a link to it in the header. Once this is deployed and working outside of common, of course.